nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryVersioning.java

```diff
@@ -423,8 +425,10 @@ public void run() {
                     nextTransaction();
                     DocumentModel restored = session.getDocument(docRef);
                     assertEquals("t1", restored.getPropertyValue("dc:title"));
-                } catch (InterruptedException | BrokenBarrierException | TimeoutException | RuntimeException
-                        | AssertionError t) {
+                } catch (InterruptedException t) {
+                    Thread.currentThread().interrupt();
+                    throwables[0] = t;
```

Comment:
Should that be 0 or 1?  In the previous section you use 0 but below this you use 1.

Reply:
Thanks, it's a copy/paste typo.